taker-frontend/src/components/LiquidationPrice.ts

```diff
@@ -0,0 +1,16 @@
+function quantoLiquidationPrice(leverage: number, quantity: number, price: number) {
```

Comment:
🔧 Can we rename these functions? In `daemon` I added the suffix `shift`, because it is by how much the initial price has to shift before you get liquidated.